docs/user-guide/building-assistants.rst

```diff
@@ -313,22 +336,34 @@ You can read more in this `blog post <https://blog.rasa.com/response-retrieval-m
 `Retrieval Actions <https://rasa.com/docs/rasa/core/retrieval-actions/>`_ page.
 
 Using the features we described in this tutorial, you can easily build a context-less assistant.
-When you’re ready to enhance your assistant with context, check out :ref:`build-contextual-assistant`.
+When you’re ready to enhance your assistant with context, check out :ref:`tutorial-contextual-assistants`.
+
+
+.. note::
+    Here's a minimal checklist of what you need to build a basic FAQ assistant:
+
+      - ``data/nlu.md``: NLU training data for ``faq/`` intents
+      - ``data/responses.md``: responses associated with ``faq/`` intents
+      - ``config.yml``: ``ReponseSelector`` in your NLU pipeline
+      - ``domain.yml``: a retrieval action ``respond_faq`` and intent ``faq``
+      - ``data/stories.md``: a simple story for FAQs
+      - ``test_stories.md``: E2E test stories for your FAQs
```

Comment:
Can we make this more like "Create NLU training data for faq/ intents in your nlu.md file" - so it's more narrative, rather than just a list of files

Reply:
For a checklist doesn't it make sense to have it by file? since the tutorial is narrative already, it might be helpful to have it like this at the end. Also in the other sections, there's multiple changes per file, and it was a way to organise the changes

Reply:
that makes sense yeah, but then maybe we should say "These are the files we modified during this tutorial" or something similar. Rather than saying "these are the steps we did". Then i'm ok with files :D

Reply:
Yeah that also makes sense, will change the wording